### DISCUSSION TITLE
Image is not loaded/selected at Power up

### DISCUSSION THREAD

**Discussion by jurrrrrr** [keirf/flashfloppy#529] [Q&A] [answered]
Hi,
My Gotec does not load/select any image at power up. The 3-digit display keeps on showing F-F. After power up i always have to manually eject and then put the USB disk in again before it starts to show disk image numbers (it then works fine). The FF.cgf has got the startup/initialisation settings: ejected-on-startup = no  and image-on-startup = last. I am using FF v3.28 on a Gotek 3 digit model (sfrkc30.at4.7). Does any one has suggestions so that it does load/select an image at power up?
Thanks in advance.

Jur

**Comment by keirf** [comment, marked as answer]
It may be related to the specific USB stick. Does a different one work okay?

**Comment by jurrrrrr** [comment]
I used a different USB stick and yes it works fine now. Many thanks!
